profiles/arch/x86/package.use.force

```diff
@@ -1,6 +1,12 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Arthur Zamarin <arthurzam@gentoo.org> (2022-01-16)
+# Bootstrapping is possible using our build snapshot, but force
+# the user to rebuild
+dev-java/openjdk:11 -system-bootstrap jbootstrap
+dev-java/openjdk:17 -system-bootstrap jbootstrap
+
```

Comment:
let's add
`REQUIRED_USE="!system-bootstrap? ( jbootstrap )"`  in the ebuild.

and maybe don't use.force `jbootstrap` here, it's already enabled with `+` in ebuild.